app/ide-desktop/lib/content/src/index.ts

```diff
@@ -195,16 +195,16 @@ class Main implements AppRunner {
             },
         })
 
-        // We override the remote logger stub with the "real" one. Eventually the runner should not be aware of the
-        // remote logger at all, and it should be integrated with our logging infrastructure.
+        // We override the remote logger stub with the "real" one. Eventually the runner should not
+        // be aware of the remote logger at all, and it should be integrated with our logging infrastructure.
         const remoteLogger = accessToken != null ? new remoteLog.RemoteLogger(accessToken) : null
-        newApp.remoteLog = async (message: string, metadata: unknown) => {
+        newApp.remoteLog = (message: string, metadata: unknown) => {
             const metadataObject =
                 typeof metadata === 'object' && metadata != null ? metadata : { metadata }
             const actualMetadata =
                 loggingMetadata == null ? metadata : { ...loggingMetadata, ...metadataObject }
             if (newApp.config.options.dataCollection.value && remoteLogger != null) {
-                await remoteLogger.remoteLog(message, actualMetadata)
+                // await remoteLogger.remoteLog(message, actualMetadata)
```

Comment:
please add properly formated fixme with this issue linked `https://github.com/enso-org/cloud-v2/issues/735`

Reply:
@PabloBuchu should be done now